clif/python/types.h

```diff
@@ -36,7 +36,7 @@ headers are included.
 #include <type_traits>
 // 
 // CLIF use `::proto2::Message` as proto2_Message
-#include "clif/python/pyproto.h"
+// #include "clif/python/pyproto.h"
```

Comment:
Could you add a comment here saying this line was commented out because we don't need CLIF's protobuf wrapping functionality at the moment and we don't want PyKaldi to depend on the availability of protobuf headers.